CarRentalService.Tests/Fixture/Fixture.cs

```diff
@@ -0,0 +1,209 @@
+﻿using CarRentalService.Domain.Domain;
+
+namespace CarRentalService.Tests.Fixture;
+
+/// <summary>
+/// Provides test data for car rental service
+/// </summary>
+public class CarRentalFixture
+{
+    /// <summary>
+    /// Collection of vehicle models
+    /// </summary>
+    public List<VehicleModel> Models { get; } = new();
```

Comment:
Упростить инициализацию коллекции тут и по коду ниже.
```suggestion
    public List<VehicleModel> Models { get; } = [];
```